src/voodoo/spellbook/spells/haste.ts

```diff
@@ -27,7 +27,7 @@ export const haste: SpellFunction = async (voodoo, accountId, upgradeConfigs) =>
     }
   });
 
-  const multiplier = 1 + attributes.intensify / 100;
+  const multiplier = attributes.intensify / 100;
```

Comment:
Shouldn't call this a multiplier anymore if you remove the `1`.

```suggestion
  const bonus = attributes.intensify / 100;
```